### PR TITLE
MC7000 - describe changes for Mixxx 2.3.1

### PR DESCRIPTION
**Pull request by toszlanyi**
correlates to https://github.com/mixxxdj/mixxx/pull/4021

### CODE REVIEW

File: source/hardware/controllers/denon_mc7000.rst
Comment (ronso0) [inline source/hardware/controllers/denon_mc7000.rst:155]:
```suggestion
   "52", ":hwlabel:`FILES`",                          "Press to maximize the library. Press this button again to exit maximized library.
```
z for consistency ; )
here and below

File: source/hardware/controllers/denon_mc7000.rst
Comment (ronso0) [inline source/hardware/controllers/denon_mc7000.rst:188]:
```suggestion
   If the library is maximized (use :hwlabel:`FILES` button) turning the jog wheel will scroll the current table view (currently playing track is not affected). Be careful though: after exiting the maximized library view the jog wheel will immediately return to jog or scratch mode."
```

File: source/hardware/controllers/denon_mc7000.rst
Comment (ronso0) [inline source/hardware/controllers/denon_mc7000.rst:227]: press Shift + Censor _OR_ press Shift + turn wheel?
If so this needs to be clarified IMO

**Review by ronso0 — CHANGES_REQUESTED** [review]

File: source/hardware/controllers/denon_mc7000.rst
Comment (toszlanyi) [reply source/hardware/controllers/denon_mc7000.rst:227]: Shift + jog wheel is a side effect from seeking through a track. Maybe it's better I just delete the comment and only mention the dedicated backspin function on the censor button. Gonna send an update later today. Cheers!

File: source/hardware/controllers/denon_mc7000.rst
Comment (toszlanyi) [reply source/hardware/controllers/denon_mc7000.rst:188]: done

File: source/hardware/controllers/denon_mc7000.rst
Comment (toszlanyi) [reply source/hardware/controllers/denon_mc7000.rst:155]: done

File: source/hardware/controllers/denon_mc7000.rst
Comment (ronso0) [inline source/hardware/controllers/denon_mc7000.rst:210]: This sounds like an interesting feature!

File: source/hardware/controllers/denon_mc7000.rst
Comment (ronso0) [inline source/hardware/controllers/denon_mc7000.rst:333]:
```suggestion
   To use default Mixxx behaviour and allow playing multiple samplers at the same time you can set the user variable ``MC7000.prevSamplerStop`` inside the :ref:`JavaScript file<denon_mc7000_uservariables>` to ``false``.
```

**Review by ronso0 — CHANGES_REQUESTED** [review]

File: source/hardware/controllers/denon_mc7000.rst
Comment (ronso0) [reply source/hardware/controllers/denon_mc7000.rst:333]: That's it from me. LGTM when the line above is changed!
Someone who reviewed the actual mapping should take a look though.

Comment (toszlanyi) [reply source/hardware/controllers/denon_mc7000.rst]: done

File: source/hardware/controllers/denon_mc7000.rst
Comment (toszlanyi) [reply source/hardware/controllers/denon_mc7000.rst:210]: Yes it is. There was a discussion on [zulip chat](https://mixxx.zulipchat.com/#narrow/stream/109122-general/topic/Slip.20Mode/near/240571349) that triggered me to add it to the MC7000 mapping. Short timer after the release to enable SLIP again. 50ms seems to be very good. Short enough you could not press a button (incl platter) twice that quickly. Long enough not to miss something. The 50ms is a pure empiric value though

**Comment by Be-ing** [comment]
@ronso0 please merge if this is ready and let's get 2.3.1 released.

**Comment by ronso0** [comment]
oh, for some reason I did not get notified about your post.
LGTM, thank you!

**Pull request merged**
